### PR TITLE
Libraries and Contributing page

### PR DESCRIPTION
**Pull request by jwcooper**
Work in progress...

### CODE REVIEW

File: libraries/index.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    <p>
+      The libraries in each release are compiled to .mpy for all recent major versions
+      of CircuitPython. Please download the bundle that matches the major version
+      of your CircuitPython. For example, if you are running 4.0.0 you should
+      download the 4.x bundle. The .mpy files are designed to take up less space on
+      your CIRCUITPY drive. Unless you have a specific reason, choose the appropiate
+      .mpy library bundle!
+    </p>
+    <p>
+      To install, download the appropriate bundle for your version of CircuitPython.
```
Comment (kattni) [inline libraries/index.html]: I'd like to make "To install" bold. It sort of disappears into the other paragraphs as it is.

Comment (jwcooper) [reply libraries/index.html]: Should be good to go there now.

File: libraries/index.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        <h3>Bundle Version 3.x</h3>
+        <p>
+          This bundle is built for use with CircuitPython 3.x.x. If you are using
+          CircuitPython 3, please download this bundle.
+        </p>
+      </div>
+      <div id="adafruit-circuitpython-bundle-py">
+        <h3>Bundle Version py</h3>
+        <p>
+          This bundle is the uncompiled source code for every library. This bundle is
```
Comment (kattni) [inline libraries/index.html]: Please change the text here to the following:
```
This bundle is the uncompiled source code for every library. It is not
intended for general use! It is only recommended if you need to edit
a library file. This bundle works with all versions of CircuitPython.
```
It is currently very redundant. (Which is obviously on me. I didn't catch it until last night.)

Comment (jwcooper) [reply libraries/index.html]: Completed and pushed.

File: libraries/contributing.html [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+              {% endfor %}
+           {% endfor %}
+           </ul>
+         </li>
+       {% endfor %}
+       </ul>
+    </div>
+    <h2>Repo-Level Issues</h2>
+    <p>
+      The following are further issues we've identified with the library repositories. Most of these
```
Comment (kattni) [inline libraries/contributing.html]: Please change this paragraph to:
```
      The following are issues with the library infrastructure. Having a standard library structure 
      greatly improves overall maintainability. Accordingly, we have a series of checks to ensure 
      the standard is met. Most of these are changes that can be made via a pull request, however 
      there are a few checks reported here that require changes to GitHub settings. If you are 
      interested in addressing any of these issues, please feel free to contact us with any questions.
```

File: libraries/index.html [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+      started. Check out the Contributing page for detailed information!
+    </p>
+    <p>
+      Find out more about how you can
+      <a href="{{ "/libraries/contributing" | relative_url  }}" title="Contributing Page">contribute to CircuitPython libraries</a>.
+    </p>
+    <!-- (can we even include the date like this?) -->
+    <h2>Library Activity for Week of  {{ site.data.libraries.updated_at | date: "%a, %b %d, %Y" }}</h2>
+    <p>
+      Thank you to this week's contributors:
```
Comment (kattni) [inline libraries/index.html]: I'd like this to be a standalone line: `Thank you to all of our contributors!`

We're going to add the number of pull requests merged to the json file. Then I would like the following line followed by two bullet points:
```
This week there were [x] pull requests merged:
* Authors: [contributor names]
* Reviewers: [reviewer names]
```
The list of reviewers is also contained in the json file.

**Comment by jwcooper** [comment]
The last change needed based on feedback is the [x] count for the pull requests merged added to libraries.json

File: libraries/index.html [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+        <li>
+          Reviewers:
+          {% for reviewer in site.data.libraries.reviewers %}
+            <a href="https://github.com/{{reviewer}}" title="Contributor Name">{{reviewer}}</a>{% if forloop.last != true %}, {% endif %}
+          {% endfor %}
+        </li>
+      </ul>
+    </p>
+    <p>
+      These are the new and updated libraries from the past week.
```
Comment (kattni) [inline libraries/index.html]: Apologies for forgetting! I meant to have you take this line out. It looks odd tucked in there. The title "Library Activity etc" followed by the headers "New Libraries" and "Updated Libraries" is clear without it.

**Comment by jwcooper** [comment]
@kattni, @sommersoft this is ready to go, if we're ready to merge the script. I think we could merge the script to master first, and then merge this in after.

**Comment by kattni** [comment]
Discussed with both @jwcooper and @sommersoft. This is ready to merge!

**Pull request merged**